Sources/MapboxMaps/Annotations/Generated/PointAnnotation.swift

```diff
@@ -38,19 +38,24 @@ public struct PointAnnotation: Annotation, Equatable {
         set { gestureHandlers.value.longPress = newValue }
     }
 
-    public var dragBeingHandler: (() -> Void)? {
-        get { gestureHandlers.value.dragBeing }
-        set { gestureHandlers.value.dragBeing = newValue }
+    /// Handles drag gesture beging.
+    ///
+    /// Should return `true` if the gesture is handled, or `false` to propagate it to the annotations or layers below.
+    public var dragBeginHandler: ((MapContentGestureContext) -> Bool)? {
```

Comment:
Noting that this technically breaks public API, semver will requre a new major release.

Reply:
@azarovalex That's fine, this feature isn't released yet, it only lives in the branch.